static/_redirects

```diff
@@ -30,6 +30,8 @@
 /docs/api-reference/labels-annotations-taints/     /docs/reference/labels-annotations-taints/ 301
 /docs/reference/using-api/api-overview/     /docs/reference/using-api/ 301
 /docs/reference/access-authn-authz/controlling-access/   /docs/concepts/security/controlling-access/ 301
+/docs/reference/kubectl/cheatsheet/   /docs/reference/kubectl/quick-guide/ 301
+/kubectlguide /docs/reference/kubectl/quick-guide/ 302
```

Comment:
This sounds good to me. And I don't mind if the existing `/cheatsheet` page is there, so long as the page people get to doesn't carry the implication of breaking exam rules.